pkg/controller/nodes/task/config/config.go

```diff
@@ -50,7 +50,7 @@ type BarrierConfig struct {
 }
 
 type TaskPluginConfig struct {
-	EnabledPlugins []string `json:"enabled-plugins" pflag:",deprecated"`
+	EnabledPlugins []string `json:"enabled-plugins" pflag:",Plugins enabled currently"`
```

Comment:
I'm confused... before I dig into code, do you know why are there two "EnabledPlugins" configs? one in plugins that you just modified and one here?

Reply:
I'm not sure exactly. Both the [plugins config](https://github.com/flyteorg/flyteplugins/blame/5a80da6b9d9a6fbca413148aaf712e5615be6e52/go/tasks/config/config.go#L20) and [propeller config](https://github.com/flyteorg/flytepropeller/pull/1) have been there since the initial commit. The propeller [default plugin for task type PR](https://github.com/flyteorg/flytepropeller/pull/199) change the EnabledPlugins comment to include "deprecated" but then still used the field - I wonder if the wrong repository was deprecated? I don't see the plugins config field used anywhere.

Reply:
Looks like the plugins' EnabledPlugins field is not used. Maybe we should delete that then...

Reply:
Sure, took care of it in the [flyteplugins pr](https://github.com/flyteorg/flyteplugins/pull/248).